app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/helm/BakeHelmConfigForm.tsx

```diff
@@ -193,6 +193,17 @@ export class BakeHelmConfigForm extends React.Component<IBakeHelmConfigFormProps
             />
           )}
         </StageConfigField>
+        <StageConfigField
+          fieldColumns={6}
+          helpKey={'pipeline.config.bake.manifest.helm.rawOverrides'}
```

Comment:
```suggestion
          helpKey="pipeline.config.bake.manifest.helm.rawOverrides"
```

Reply:
We don't need to wrap string literals in JSX with brackets, but I see that the rest of this file doesn't follow this convention so feel free to leave and we can address in a future commit that we don't plan to cherry-pick

Reply:
👍 good call out!